src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java

```diff
@@ -27,14 +28,18 @@ public class DispatchManager {
     private long lastDispatcherUpdate;
     private final Map<String, Integer> collected;
     private final Map<String, Integer> lastCollected;
-    private final Gson g;
+    private final Gson gson;
+    private final CaptchaHandler captchaHandler;
 
     DispatchManager(BackpageManager backpageManager) {
         this.backpageManager = backpageManager;
         this.data = new DispatchData();
         this.collected = new HashMap<>();
         this.lastCollected = new HashMap<>();
-        this.g = backpageManager.getGson();
+        this.gson = backpageManager.getGson();
+        this.captchaHandler = new CaptchaHandler(backpageManager,
+                backpageManager.main.configHandler.requireConfig("config.miscellaneous.solve_backpage_captcha"),
```

Comment:
```suggestion
                configApi.requireConfig("config.miscellaneous.solve_backpage_captcha"),
```
pass in the `ConfigAPI configApi` in the constructor, make BackpageManager also 
 have it on the constructor. It should get auto-injected if i'm not mistaken, that avoids you needing to have all this chain of dependencies